tests/baselines/reference/docker/vscode.log

```diff
@@ -3,22 +3,22 @@ Standard output:
 yarn run vX.X.X
 $ node --max_old_space_size=4095 ./node_modules/gulp/bin/gulp.js compile
 [XX:XX:XX] Using gulpfile /vscode/gulpfile.js
-[XX:XX:XX] [monaco.d.ts] monaco.d.ts is changed - total time took ?s
+[XX:XX:XX] Error: /vscode/extensions/html-language-features/server/src/modes/javascriptMode.ts(130,36): Expected 7 arguments, but got 6.
```

Comment:
Broken in https://github.com/microsoft/TypeScript/pull/42890; the associated fixup PR https://github.com/microsoft/vscode/pull/117092 doesn't cover this line of code.

@andrewbranch can you look at this?

Reply:
https://github.com/microsoft/vscode/pull/120362